File: C99/1151.c
#include <stdio.h>

int main(){
    int inp,n1 = 0,n2 = 1,next=0;
    
    scanf("%i",&inp);
    
    for(int i = 1;i <= inp;i++){
        if(i==inp){
            printf("%i\n",n1);
        }else{
            printf("%i ",n1);
            next=n1+n2;
            n1=n2;
            n2=next;
        }
    }

    return 0;
}

File: javascript/1151.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var inp = parseInt(input);
var n1 =0, n2 = 1, next = 0;
var values = [];
for(let i = 1; i <= inp; i++){
    if(i == inp){
        values.push(n1);
    }else{
        values.push(n1);
        next = n1 + n2;
        n1 = n2;
        n2 = next;
    }
}

console.log(values.join(" "));